### ISSUE TITLE
flash of unstyled text (on devserver only)

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#893]
Somewhere in #871 we introduced a change that causes a flash of unstyled text on load. Yuck.

**Comment by indirectlylit** [comment]
looks like the fonts get stalled for over a second:

![image](https://cloud.githubusercontent.com/assets/2367265/22986969/1cd18a94-f362-11e6-88ff-270a6f41b035.png)

They are currently base-64 in the css, which might be part of the issue:

```
@font-face {
  font-family: 'NotoSans';
  src: url(data:application/font-woff;charset=utf-8;base64,d09GRgABAAAAA...
```

**Comment by christianmemije** [comment]
Does this also happen on a build version, or just the dev-server?

**Comment by indirectlylit** [comment]
no, it does not seem to happen in production builds. Is there something different about our webpack production config which prevents the issue?

**Comment by christianmemije** [comment]
Well all the stylesheets are `<link` and reference a blob.

**Comment by rtibbles** [comment]
This should have been fixed in #1737
